src/main/kotlin/calculator/StringAddCalculator.kt

```diff
@@ -21,28 +21,24 @@ class StringAddCalculator {
     private fun getValues(value: String): List<String> {
         val result = Regex(CUSTOM_DELIMITER).find(value)
         result?.let {
-            val customDelimiter = it.groupValues[1]
-            return it.groupValues[2].split(customDelimiter)
+            val customDelimiter = it.groupValues[CUSTOM_DELIMITER_INDEX]
+            return it.groupValues[CUSTOM_DELIMITER_EXPRESSION_INDEX].split(customDelimiter)
         }
 
         return value.split(DEFAULT_DELIMITER.toRegex())
     }
 
-    private fun validateWithConvertZero(text: String?): String {
-        if (text.isNullOrEmpty()) {
-            return ZERO
-        }
-
-        if (text.isBlank()) {
-            return ZERO
-        }
-
-        return text
+    private fun validateWithConvertZero(text: String?): String = when {
+        text.isNullOrEmpty() -> ZERO
+        text.isBlank() -> ZERO
```

Comment:
```suggestion
        text.isNullOrEmpty(),
        text.isBlank() -> ZERO
```
같은 값을 리턴하는 경우에는 콤마로 묶을 수 있어요

Reply:
컴마를 사용하니 사용할수 없다고하는데요 ㅠ 혹시 코틀린 버전에 대한 이슈도 있을 수 있을까요?

Reply:
제가 잘못보고 리뷰드렸습니다 😭  🙏